cli/tests/pcluster/config/defaults.py

```diff
@@ -126,6 +126,7 @@
     "raid_settings": None,
     "fsx_settings": None,
     "dcv_settings": None,
+    "enable_intel_hpc_platform": False,
```

Comment:
Please move it before the *_settings parameters

Reply:
done